### PR TITLE
security/stunnel: Add client mode option to services

### PR DESCRIPTION
**Pull request by topperblues**
### Problem
I need to connect to an LDAP server using stunnel client mode.
In the services form, there is no option to enable client mode.

### Solution
This adds a checkbox to enable client mode in the service creation/edit form.
When the checkbox is enabled, the option
`client = yes`
is added to relative service config.

### CODE REVIEW

File: security/stunnel/src/opnsense/mvc/app/models/OPNsense/Stunnel/Stunnel.xml [hunk 1]
```diff
@@ -63,6 +63,10 @@
                     <default>0</default>
                     <Required>Y</Required>
                 </enableCRL>
+                <clientmode type="BooleanField">
+                    <default>0</default>
+                    <Required>Y</Required>
```
Comment (AdSchellevis) [inline security/stunnel/src/opnsense/mvc/app/models/OPNsense/Stunnel/Stunnel.xml:68]: @topperblues the model needs a version upgrade for this to work.

**Comment by AdSchellevis** [comment]
@topperblues I totally forgot about this, sorry about that. If you increase the model version in this PR, I'll pull it in.

**Comment by topperblues** [comment]
> @topperblues I totally forgot about this, sorry about that. If you increase the model version in this PR, I'll pull it in.

Model version updated, thank you!

**Comment by AdSchellevis** [comment]
@topperblues thanks!

**Pull request merged**